src/js/view/controls/components/timeslider.js

```diff
@@ -108,9 +108,17 @@ class TimeSlider extends Slider {
 
         // Show the tooltip on while dragging (touch) moving(mouse), or moving over(mouse)
         this.elementUI = new UI(this.el, { useHover: true, useMove: true })
-            .on('drag move over', this.showTimeTooltip.bind(this), this)
-            .on('dragEnd out', this.hideTimeTooltip.bind(this), this)
+            .on('drag move over', this.showTimeTooltip, this)
+            .on('out', this.hideTimeTooltip, this)
             .on('click', () => this.el.focus());
+
+        this.el.addEventListener('focus', (e) => this.updateAriaText(e));
```

Comment:
The event argument `e` is not being used in `updateAriaText` and should be removed here.